### ISSUE TITLE
The performance paper reported is 89% mIoU on PASCAL VOC 2012

### ISSUE THREAD

**Issue by RunpeiDong** [YudeWang/deeplabv3plus-pytorch#46]
Hi. In the paper of ECCV 2018, Deeplabv3+ achieved the performance of 89% of mIoU on PASCAL VOC dataset. But you claimed that the performance in paper is far lower(about 10%), do you just made a mistake of not seeing it right? By the way, the benchmark is far higher than the code can do, can you solve it?

**Comment by YudeWang** [comment]
@Andersonann 
Because 89% is achieved by COCO+JFT pretraining, output stride=8.
JFT is Google private dataset. My experiments can not use output stride=8 setting with only 4 GPUs.

**Comment by RunpeiDong** [comment]
Thank you! I see. So have you tried to follow Deeplab v3+’s training step to test if can get the performance? Have you tried to use gradient accumulation trick to make up for the GPU’s memory issue. By the way, how can I get JFT dataset and, can I? Since it is google private? Thank you again.
(Are u Chinese? If you are, maybe we can add Wechat to talk?)


------------------ Original ------------------

**Comment by YudeWang** [comment]
Gradient accumulation trick is not used.
You can get JFT by joining Google :)

**Comment by RunpeiDong** [comment]
Cool, thanks again.
